### PR TITLE
chore: improve readme

### PR DESCRIPTION
**Pull request by TAdev0**
# Pull Request type

- Documentation content changes

## Does this introduce a breaking change?

No

### CODE REVIEW

**Comment by TAdev0** [comment]
Hi @antiyro , 

updated readme after running some issues following the readm guide. 
Btw, do you know what is the current size in GB of a Deoxys/Madara full node synchronized from block 1?

File: README.md [hunk 1]
```diff
@@ -58,19 +58,19 @@ Madara is a powerfull Starknet hybrid client written in Rust.
    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s
    ```
 
-   Clone the Madara repository:
+   Clone the Deoxys Madara full node repository:
```
Comment (antiyro) [inline README.md]: could you keep the naming here

Comment (antiyro) [reply README.md]: we are migrating toward madara so anything that doesnt technically affect the code or its deployement should be under madara namings

File: README.md [hunk 2]
```diff
@@ -58,19 +58,19 @@ Madara is a powerfull Starknet hybrid client written in Rust.
...
 
    ```sh
    cd <your-destination-path>
-   git clone https://github.com/KasarLabs/madara .
+   git clone https://github.com/KasarLabs/deoxys
```
Comment (antiyro) [inline README.md]: here is fine

Comment (TAdev0) [reply README.md]: ok got it! thanks

**Comment by TAdev0** [comment]
@antiyro  pr updated

**Comment by antiyro** [comment]
Please ensure you add changelogs for the CI

**Comment by TAdev0** [comment]
done

**Review by antiyro — APPROVED** [review]

**Pull request merged**